src/toil_scripts/rnaseq_cgl/README.md

```diff
@@ -141,3 +152,73 @@ rev-3pr-adapter: AGATCGGAAGAG
 To run on a distributed AWS cluster, see [CGCloud](https://github.com/BD2KGenomics/cgcloud) for instance provisioning, 
 then run `toil-rnaseq run aws:us-west-2:example-jobstore-bucket --batchSystem=mesos --mesosMaster mesos-master:5050`
 to use the AWS job store and mesos batch system. 
+
+# Methods
+
+## Tools
+
+| Tool     | Version | Description                                                                                                                                 |
+|----------|---------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| FastQC   | 0.11.5  | Obtains quality metrics on each FASTQ input file.                                                                                           |
+| CutAdapt | 1.9     | Adapter trimming and quality checking by enforcing fastq samples are properly paired.                                                       |
+| STAR     | 2.4.2a  | Aligns fastq samples to the genome. Produces transcriptome bam for RSEM, and can optionally generate a genome-aligned bam and BigWig files. |
+| RSEM     | 1.2.25  | Performs quantification of RNA-seq data to produces count values for genes and isoforms.                                                    |
+| Kallisto | 0.42.4  | Performs quantification of RNA-seq data to produces counts for isoforms directly from fastq data.                                           |
+
+
+## Reference Data
+
+HG38 (no alternative sequences) was downloaded from [NCBI](ftp://ftp.ncbi.nlm.nih.gov/genomes/archive/old_genbank/Eukaryotes/vertebrates_mammals/Homo_sapiens/GRCh38/seqs_for_alignment_pipelines/).
+The PAR locus on the Y chromosome, which has duplicate sequences relative to the X chromosome, were removed. chrY:10,000-2,781,479
```

Comment:
Is this done by the pipeline itself, or in your reference file build?

Also, might be worthwhile to explain why/link to a publication that explains why. However, this might be sufficiently obvious to not need to be explained. I defer to you.